README.md

```diff
@@ -13,3 +13,11 @@ Astro.
 
 Please see the [`@apostrophecms/apostrophe-astro`](https://github.com/apostrophecms/apostrophe-astro) documentation
 for complete information about how to get started with this project.
+
+## Postcss plugin
+
+As you can see, there is a postcss config adding one plugin.
```

Comment:
```suggestion
As you can see, there is a postcss config adding one plugin.
```
The [ApostropheCMS breakpoint preview feature](https://docs.apostrophecms.org/guide/pages.html#device-breakpoint-previews) allows you to preview how your site appears on different devices by generating temporary CSS for media queries. This functionality relies on a PostCSS plugin. While you can customize the `postcss.config.js` file for your project, ensure this plugin remains included to maintain proper functionality of the feature.